jumpscale/clients/stellar/stellar.py

```diff
@@ -20,7 +20,7 @@
 
 
 _THREEFOLDFOUNDATION_TFTSTELLAR_SERVICES = {"TEST": "testnet.threefold.io", "STD": "tokenservices.threefold.io"}
-_HORIZON_NETWORKS = {"TEST": "https://horizon-testnet.stellar.org", "STD": "https://horizon.stellar.org"}
+_HORIZON_NETWORKS = {"TEST": "http://stellar-watcher.grid.tf", "STD": "https://horizon.stellar.org"}
```

Comment:
shouldn't the domain name include an indication of testnet? also shouldn't it be https?

Reply:
updated here: c86762c